Makefile

```diff
@@ -175,31 +175,46 @@ install: manifests ## Install CRDs into the K8s cluster specified in ~/.kube/con
 uninstall: manifests ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	kubectl delete -k $(KUSTOMIZE_CONFIG_CRD) --ignore-not-found=$(ignore-not-found)
 
-KUSTOMIZE_CONFIG_DEFAULT ?= config/default
-KUSTOMIZE_CONFIG_HUB_DEFAULT ?= config/default-hub
+KUSTOMIZE_CONFIG_CERT_MANAGER ?= https://github.com/cert-manager/cert-manager/releases/download/v1.11.0/cert-manager.yaml
+
+.PHONY: deploy-cert-manager
+deploy-cert-manager:
+	kubectl apply -f $(KUSTOMIZE_CONFIG_CERT_MANAGER)
+	kubectl -n cert-manager wait --for=condition=Available deployment \
```

Comment:
We don't need to wait for the cert-manager operator to be ready - we just need to wait for the CRDs to have the `Established` condition.

Reply:
I think there is also an http call done and if the pod/service isn't ready we get a timeout. I will need to validate it though.

Reply:
`cert-manager` will only generate the `Secret` KMM needs once it's ready. Thus it may take some time for the KMM webhook to come online; in the meantime all `Module` creations and updates will fail. I think that is fine; if a user wants to wait for KMM (and thus `cert-manager`) to be ready, they can wait for the KMM `Deployment` to be `Available`.

Reply:
So the first error we may hit is the CRD not installed:
```
resource mapping not found for name: "kmm-operator-serving-cert" namespace: "kmm-operator-system" from "config/default": no matches for kind "Certificate" in version "cert-manager.io/v1"
ensure CRDs are installed first
resource mapping not found for name: "kmm-operator-selfsigned-issuer" namespace: "kmm-operator-system" from "config/default": no matches for kind "Issuer" in version "cert-manager.io/v1"
ensure CRDs are installed first
make: *** [Makefile:185: deploy] Error 1
```

And then you can get another (later) error:
```
Error from server (InternalError): error when creating "config/default": Internal error occurred: failed calling webhook "webhook.cert-manager.io": failed to call webhook: Post "https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=10s": dial tcp 10.96.229.242:443: connect: connection refused
Error from server (InternalError): error when creating "config/default": Internal error occurred: failed calling webhook "webhook.cert-manager.io": failed to call webhook: Post "https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=10s": dial tcp 10.96.229.242:443: connect: connection refused
make: *** [Makefile:185: deploy] Error 1
```

As I understand it is is the Certificate webhook validation that isn't ready yet.
So I don't see another option rather than waiting for the webhook pod to be ready (which is the slowest of all tree so there is no harm in waiting to all of the cert-manager deployments).

Reply:
I did not see those in my deployment. Totally makes sense to wait for the deployment as well indeed.